opentok/opentok-meet
# OPENTOK-47711: show captions from multiple participants

#### What is this PR doing?
Adds captions for up to four participants on the screen at once (this number is configurable). Also adds a timeout for users that haven't spoken recently and updates some CSS. Most of the code changes live in https://github.com/michaelwhittemore/OpenTok-Angular/pull/2/files, but we don't currently have a master angular branch. 

#### How should this be manually tested?
Join a captioned sessions with several participants and ensure that you can see everyone's text as they speak.  

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-47711
